platforms/jvm/language-java/src/main/java/org/gradle/api/internal/tasks/compile/DiagnosticToProblemListener.java

```diff
@@ -132,14 +141,14 @@ void printDiagnosticCounts() {
      * @param kind the kind of diagnostic (error, or warn)
      * @param number the total number of diagnostics of the given kind
      */
-    private static void printDiagnosticCount(Log logger, String kind, int number) {
+    private static String diagnosticCount(Log logger, String kind, int number) {
         // Compiler only handles 'error' and 'warn' kinds
         if (!("error".equals(kind) || "warn".equals(kind))) {
             throw new IllegalArgumentException("kind must be either 'error' or 'warn'");
         }
         // If there are no diagnostics of this kind, we don't need to print anything
         if (number == 0) {
-            return;
+            return "";
```

Comment:
🤔 Might become `null` based on the suggestion above.
Please resolve this if not interested.

Reply:
Fixed, ptal